metrics/example/metrics.py

```diff
@@ -1,26 +1,32 @@
 import json
-import math,sys,time
+import math,os,time
 import random
 import requests
 import datetime
 
 print("*******************Custom Metrics Generator***************************")
 url = "http://dkube-exporter.dkube:9401/modelmonitor/metrics"
 monitor_id = os.getenv("MM_ID")
-print(monitor_id)
 run_id = os.getenv("MM_RUNID")
-
-with open(MM_CONFIG_FILE) as f:
+performance_frequency = 0
+with open(os.getenv("MM_CONFIG_FILE")) as f:
     configuration = json.load(f)
-
-metric_value = random.random()
+    if configuration:
+        drift_monitoring = configuration.get("performance_monitoring")
```

Comment:
You mean performance_monitoring in place of drift_monitoring for the variable name so it is not confusing?

Reply:
yeah variable name is incorrect